src/components/task-step/step-footer-mixin.cjsx

```diff
@@ -20,16 +22,30 @@ module.exports =
 
     taskDetails
 
+  renderCoversSections: (sections) ->
+    sections = _.map sections, (parts) ->
+      combined = parts.join('.')
```

Comment:
Heh, I didn't know about the `<ChapterSection>` component.  Looking at it now, but I think it'll have to be reworked to use `join` since Bio has 3 section components. i.e. `1.1.2`